src/thirdparty/font-awesome-svg-png/white/svg/circle.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1792 1792"><circle cx="50%" cy="50%" r="42.9%" fill="#fff"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1664 896q0 209-103 385.5t-279.5 279.5-385.5 103-385.5-103-279.5-279.5-103-385.5 103-385.5 279.5-279.5 385.5-103 385.5 103 279.5 279.5 103 385.5z" fill="#fff"/></svg>
```

Comment:
Maybe the problem is that something expected a `path` and not a `circle`? In any case, the path can be shorter:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17920 17920"><path fill="#fff" d="M16640 8960q0 2090-1030 3855t-2795 2795-3855 1030-3855-1030-2795-2795-1030-3855 1030-3855 2795-2795 3855-1030 3855 1030 2795 2795 1030 3855z"/></svg>
```

Reply:
Yeah that could be it. I think the build script assumes a path? In any case I saw the bug in both Firefox and Chrome, so I don't think it's a browser issue.

Reply:
You're right. OK that is a much simpler fix.